NOTICES.txt

```diff
@@ -360,200 +500,873 @@ TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
 
-       b) use the modified software only within your corporation or
-          organization.
 
-       c) give non-standard binaries non-standard names, with
-          instructions on where to get the original software distribution.
+--------------- SECTION 2: BSD 3-clause "New" or "Revised" License ----------
 
-       d) make other distribution arrangements with the author.
+BSD 3-clause "New" or "Revised" License is applicable to the following component(s).
+
+>>> https://rubygems.org/gems/puma/versions/3.12.2
+
+BSD 3-Clause License
+
+Copyright (c) 2019, Evan Phoenix. Some code by Zed Shaw, (c) 2005.
+All rights reserved.
 
-  3. You may distribute the software in object code or binary form,
-     provided that you do at least ONE of the following:
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
 
-       a) distribute the binaries and library files of the software,
-	  together with instructions (in the manual page or equivalent)
-	  on where to get the original distribution.
+1. Redistributions of source code must retain the above copyright notice, this
+   list of conditions and the following disclaimer.
 
-       b) accompany the distribution with the machine-readable source of
-	  the software.
+2. Redistributions in binary form must reproduce the above copyright notice,
+   this list of conditions and the following disclaimer in the documentation
+   and/or other materials provided with the distribution.
 
-       c) give non-standard binaries non-standard names, with
-          instructions on where to get the original software distribution.
+3. Neither the name of the copyright holder nor the names of its
+   contributors may be used to endorse or promote products derived from
+   this software without specific prior written permission.
 
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+>>> https://rubygems.org/gems/pg/versions/0.20.0
```

Comment:
Looks like this one is actually `BSD 2-clause "Simplified" License`: https://github.com/ged/ruby-pg/blob/v0.20.0/LICENSE

Reply:
He has his own conditions or the BSD 2-clause. Would you like me to get rid of his conditions and simply replace them with the BSD 2-clause or also keep his conditions and then appen the BSD 2-clause at the bottom